src/main/java/svc/models/SitemapUrl.java

```diff
@@ -0,0 +1,24 @@
+package svc.models;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+@JacksonXmlRootElement (localName="url")
```

Comment:
It is cleaner to have it as a url element.  Then we can add priority if we need to.